### PR TITLE
Candidates post endpoint

### PR DESCRIPTION
**Pull request by heron2014**
### please merge this PR after merging [298](https://github.com/FAC-GM/app/pull/298)
Related to this issue #288 

@nelsonic please have a look at this PR, thank you.

### CODE REVIEW

File: test/job.test.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+      expect(err).to.not.exist();
+      var options = {
+        method: "POST",
+        url: "/jobs/create"
+      };
+
+
+      server.inject(options, function (res) {
+        expect(res.statusCode).to.equal(200);
```
Comment (nelsonic) [inline test/job.test.js]: can we write an expectation for some specific reply?

Comment (SimonLab) [inline test/job.test.js]: We don't know yet the format of the payload received, so at the moment we just save in es what the post request is getting and that's why we just have one test

**Comment by nelsonic** [comment]
@heron2014 good code. :+1: 
please write an expectations for the result of the test so its *clear* what you are testing. 
happy to merge.

**Comment by nelsonic** [comment]
OK. Got it.

**Pull request merged**